README.md

```diff
@@ -9,22 +11,38 @@ Issues and feature requests posted to GitHub are not stored in any
 publicly available location, so I find that I quickly lose track of
 the things that I've sent them.
 
+## For GitHubbers
+
 If you are a GitHubber, and you would like to respond and close issues
 on this repository, please let me (or one of the other collaborators)
 know, and we'll add you as a collaborator.  Or maybe use some secret
 GitHub internal APIs to get access, if you have such things.  If you
 want to know what people are asking for, or ask for them to provide
 details about use cases etc, then head over to the issues list.
```

Comment:
Remove `etc` here and change it to `details about use cases. Then, head over to the issues list.`